packages/layout/src/steps/resolvePagination.js

```diff
@@ -19,7 +19,13 @@ const assingChildren = R.assoc('children');
 
 const getHeight = R.path(['box', 'height']);
 
-const allFixed = R.all(isFixed);
+/* All the children are fixed */
+const onlyFixedChildren = n => R.all(isFixed)(n.children);
```

Comment:
Should this function name be `allChildrenFixed` instead of `onlyFixedChildren`?

Reply:
Updated